### PR TITLE
Fix: Failing jobs

### PR DESCRIPTION
**Pull request by Meldiron**
Currently, we return exit code 1 when the existing dataset is up to date. That is a problem for GitHub since it considers such a test failed and shoots notifications at us.

This PR makes sure the job exits successfully even when our data is up to date.

Tested the edge case on a fork:
CleanShot 2022-08-29 at 09 58 25@2x

### CODE REVIEW

File: .github/workflows/import.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 on:
   schedule:
     - cron: "12 3 * * *"
+  workflow_dispatch:
```
Comment (Meldiron) [inline .github/workflows/import.yml:4]: This line allows us to trigger jobs manually if ever needed. That becomes really helpful in the future, as it will allow us to test jobs - execute them from a specific branch.

The downside might be security - a person with access to workflow dispatching might be able to get hands on some secrets. Would love approval from @eldadfux  on this. If not secure, we can revert it.

CleanShot 2022-08-29 at 09 55 38@2x

Comment (eldadfux) [reply .github/workflows/import.yml:4]: As long as forks don't get access to it, we should be fine.

Comment (Meldiron) [reply .github/workflows/import.yml:4]: Forced a push on my branch. It was successful, but it pushed to my fork. Looks like we are using some pre-defined secrets for that, so it should be all safe.

**Review by eldadfux — APPROVED** [review]
